nvflare/private/fed/server/job_runner.py

```diff
@@ -528,4 +528,3 @@ def stop_all_runs(self, fl_ctx: FLContext):
             self.stop_run(job_id, fl_ctx)
 
         self.log_info(fl_ctx, "Stop all the running jobs.")
-        self.ask_to_stop = True
```

Comment:
Add back here the:

self.ask_to_stop = True

So the server_engine stop_all_jobs() does not have to call two of the job_runner methods.

self.job_runner.stop_all_runs(fl_ctx)
self.job_runner.stop()

Reply:
Ok will do